soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1409,7 +1419,71 @@ s16 Randomizer::GetItemFromGet(RandomizerGet randoGet, GetItemID ogItemId) {
             return GI_BOTTLE;
         case RG_BOTTLE_WITH_MILK:
             return GI_MILK_BOTTLE;
-            
+
+        case RG_DEKU_TREE_MAP:
+        case RG_DODONGOS_CAVERN_MAP:
+        case RG_JABU_JABUS_BELLY_MAP:
+        case RG_FOREST_TEMPLE_MAP:
+        case RG_FIRE_TEMPLE_MAP:
+        case RG_WATER_TEMPLE_MAP:
+        case RG_SPIRIT_TEMPLE_MAP:
+        case RG_SHADOW_TEMPLE_MAP:
+        case RG_BOTTOM_OF_THE_WELL_MAP:
+        case RG_ICE_CAVERN_MAP:
+            if (GetRandoSettingValue(RSK_STARTING_MAPS_COMPASSES) < 3) {
```

Comment:
not gonna stop me from merging this, but we should probably figure out a good way to not use magic numbers when pulling from RSKs

with the boolean ones it's nbd, but it's not clear what we're checking here